mg-instamojo-for-give.php

```diff
@@ -5,23 +5,23 @@
 * @package           Instamojo for Give
 * @author            Mehul Gohil
 * @copyright         2021 Mehul Gohil <hello@mehulgohil.com>
-* @license           GPL-2.0-or-later
+* @license           GPL-3.0-or-later
 *
 * @wordpress-plugin
 *
 * Plugin Name:       MG - Instamojo for Give
 * Plugin URI:        https://wordpress.org/plugins/instamojo-for-give
 * Description:       Accept donations via Instamojo payment gateway using GiveWP.
-* Version:           1.0.0
+* Version:           1.0.1
```

Comment:
@Ankur0812 there is a constant under config/constants.php where we need to change the plugin version as well